packages/l/lato-font.json

```
@@ -4,7 +4,7 @@
   "keywords": [
     "font"
   ],
-  "license": "(MIT AND OFL-1.1)",
+  "license": "(MIT OR OFL-1.1)",
```

```suggestion
  "license": "(MIT AND OFL-1.1)",
```
`AND` is valid, let's not change license identifiers supplied by the packages